examples/simple/simple_ext1/__init__.py

```diff
@@ -1,5 +1,8 @@
 from .application import SimpleApp1
 
 
-def _jupyter_server_extension_paths():
+def _jupyter_server_extension_points():
     return [{"module": "simple_ext1.application", "app": SimpleApp1}]
+
+
+_jupyter_server_extension_paths = _jupyter_server_extension_points
```

Comment:
Are these lines still needed?

Reply:
I thought I'd keep them for backwards compatibility, I removed them now.

Reply:
I think since these examples are on the 2.x branch they can use the 2.x features.  :smile: